demo/demo-perf-client/pom.xml

```diff
@@ -1,44 +0,0 @@
-<!--
```

Comment:
why delete this project?
we always use this to test performance.

Reply:
Had Recovery